pkgs/applications/graphics/gltfpack/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, nixos
```

Comment:
You are not using this argument, so it can be removed. In the other package as well.